src/mame/machine/k573dio.cpp

```diff
@@ -216,8 +216,6 @@ WRITE16_MEMBER(k573dio_device::mpeg_start_adr_low_w)
 {
 	logerror("FPGA MPEG start address low %04x\n", data);
 	k573fpga->set_mp3_cur_adr((k573fpga->get_mp3_cur_adr() & 0xffff0000) | data); // low
-	if(is_ddrsbm_fpga)
```

Comment:
DDR Solo Bass Mix doesn't know what crypto keys 2 and 3 are at all. The game only ever references the key 1 register, so this code is unnecessary.

Reply:
I just looked at this again and realized you are using crypto_key3 in place of the crypto_idx I had before. I'll revert this change.